### PR TITLE
vi: Welcome - Running the Course translation

### PR DESCRIPTION
**Pull request by khoaxuantu**
The Vietnamese translation has been set up, so I would love to contribute. Here is the translation of *Welcome to Comprehensive Rust: (1) Running the Course*.

I have followed the [Contributing Guidelines](https://github.com/google/comprehensive-rust/blob/b38d429e498b58da52f8b23d4c7c52195994e1cc/CONTRIBUTING.md):

- [x] Edited by Poedit
- [x] Formatted by `dprint fmt`
- [x] Submitted to the CLA

### CODE REVIEW

File: po/vi.po [hunk 1]
```diff
@@ -5,31 +5,32 @@ msgstr ""
...
+"X-Generator: Poedit 3.4.2\n"
 
 #: src/SUMMARY.md src/index.md
 msgid "Welcome to Comprehensive Rust 🦀"
 msgstr "Chào mừng bạn đến với Comprehensive Rust"
 
 #: src/SUMMARY.md src/running-the-course.md
 msgid "Running the Course"
-msgstr ""
+msgstr "Hướng dẫn khóa học"
```
Comment (daivinhtran) [inline po/vi.po:21]: Nit: Capitalization.

File: po/vi.po [hunk 2]
```diff
@@ -5,31 +5,32 @@ msgstr ""
...
 
 #: src/SUMMARY.md src/running-the-course/course-structure.md
 msgid "Course Structure"
-msgstr ""
+msgstr "Cấu trúc khóa học"
```
Comment (daivinhtran) [inline po/vi.po:25]: Nit: Capitalization.

File: po/vi.po [hunk 3]
```diff
@@ -5,31 +5,32 @@ msgstr ""
...
 
 #: src/SUMMARY.md src/running-the-course/keyboard-shortcuts.md
 msgid "Keyboard Shortcuts"
-msgstr ""
+msgstr "Phím tắt"
```
Comment (daivinhtran) [inline po/vi.po:29]: Nit: Capitalization.

File: po/vi.po [hunk 4]
```diff
@@ -1837,6 +1966,11 @@ msgid ""
 "get going. Translations are coordinated on the [issue tracker](https://"
 "github.com/google/comprehensive-rust/issues/282)."
 msgstr ""
+"Nếu bạn muốn đóng góp giúp đỡ, vui lòng xem [hướng dẫn của chúng tôi]"
+"(https://github.com/google/comprehensive-rust/blob/main/TRANSLATIONS.md) để "
+"biết nên bắt đầu như thế nào. Các bản dịch được hợp tác cùng hoàn thành trên "
+"một bản [issue tracker](https://github.com/google/comprehensive-rust/"
```
Comment (daivinhtran) [inline po/vi.po:1972]: The issue is rather outdated. Can you replace it with https://github.com/google/comprehensive-rust/issues/1957 instead?

**Review by daivinhtran — COMMENTED** [review]
Thanks @khoaxuantu ! This is awesome. I left a few minor comments. The rest LGTM.

**Comment by daivinhtran** [comment]
Can you also leave a comment in this [Assignment](https://docs.google.com/spreadsheets/d/1K0jK5TbX-__NVjvCJwZXjv7Fht86vu49gpOA38UfWSQ/edit#gid=0) sheet for where you translated as well?

**Comment by khoaxuantu** [comment]
> Thanks @khoaxuantu ! This is awesome. I left a few minor comments. The rest LGTM.

Hi @daivinhtran ! I just have resolved your reviews. Please take a look again.

**Review by daivinhtran — APPROVED** [review]

**Pull request merged**